packages/db/migrations/seed.ts

```diff
@@ -31,6 +32,21 @@ const seedEveryoneGroupAsync = async (db: Database) => {
   console.log("Created group 'everyone' through seed");
 };
 
+const seedOnboardingAsync = async (db: Database) => {
+  const existing = await db.query.onboarding.findFirst();
+
+  if (existing) {
+    console.log("Skipping seeding of onboarding as it already exists");
+    return;
+  }
+
+  await db.insert(onboarding).values({
+    id: createId(),
+    step: "start",
+  });
+  console.log("Created onboarding step through seed");
```

Comment:
Shouldn't we use the logger here?

Reply:
Probably not really possible except if I add additional logic for it. Not worth it IMO